Upstream: yes
From d8ee31e0d9d59f091e0a190815157aaad98db475 Mon Sep 17 00:00:00 2001
From: Wouter van Kesteren <woutershep@gmail.com>
Date: Mon, 23 Mar 2015 18:59:10 +0100
Subject: blacklist kernels instead of whitelist

Change-Id: If354836a2309dd3c155492dd6d2469fdb5465909

diff --git a/configure.ac b/configure.ac
index b9e765b..4254c79 100644
--- a/configure.ac
+++ b/configure.ac
@@ -109,10 +109,11 @@ linux*)
 	AC_MSG_CHECKING([for supported kernel])
 	kernel_version=`uname -r | cut -d'-' -f1`
 	case "$kernel_version" in
-	3.*|2.6*)
-		AC_MSG_RESULT([$kernel_version])
+	0.*|1.*|2.0.*|2.1.*|2.2.*|2.3.*|2.4.*)
+		AC_MSG_RESULT([NO!])
+		AC_MSG_ERROR([Linux kernel version $kernel_version is not supported by pinktrace])
 		;;
-	2.5*)
+	2.5.*)
 		kernel_minor_version=`echo $kernel_version | cut -d'.' -f3`
 		if test "$kernel_minor_version" -lt 46; then
 			AC_MSG_RESULT([NO!])
@@ -121,8 +122,7 @@ linux*)
 		AC_MSG_RESULT([$kernel_version])
 		;;
 	*)
-		AC_MSG_RESULT([NO!])
-		AC_MSG_ERROR([Linux kernel version $kernel_version is not supported by pinktrace])
+		AC_MSG_RESULT([$kernel_version])
 		;;
 	esac
 	;;
-- 
cgit v0.10.2

