From df93a421ce30baeed3bdc91699ff07ca104aa365 Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Sun, 22 Nov 2015 12:23:24 -0500
Subject: [PATCH] fix building on musl

---
 dwz.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/dwz.c b/dwz.c
index 8bea643..a82ca98 100644
--- a/dwz.c
+++ b/dwz.c
@@ -19,7 +19,7 @@
 
 #include <assert.h>
 #include <errno.h>
-#include <error.h>
+#include <err.h>
 #include <fcntl.h>
 #include <getopt.h>
 #include <setjmp.h>
@@ -111,6 +111,10 @@
 # define unlikely(x) x
 #endif
 
+#ifndef error
+#define error(status, errno, ...) err(status, __VA_ARGS__)
+#endif
+
 #define obstack_chunk_alloc     malloc
 #define obstack_chunk_free      free
 
-- 
2.6.3

