From f005f84900e38d43b3cf5e08c5ca65857ab34995 Mon Sep 17 00:00:00 2001
From: Emmanuel Leblond <emmanuel.leblond@gmail.com>
Date: Sun, 9 Aug 2015 08:50:11 +0200
Subject: [PATCH] Add SO_TARGET to Makefile (see #65)

---
 Makefile | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/Makefile b/Makefile
index 54a03ba..a5a4ab2 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 #
 # iniparser Makefile
 #
-.PHONY: example libiniparser.so
+.PHONY: example
 
 # Compiler settings
 CC      ?= gcc
@@ -24,7 +24,7 @@ LDSHFLAGS = -shared -Wl,-Bsymbolic
 LDFLAGS += -Wl,-rpath -Wl,/usr/lib -Wl,-rpath,/usr/lib
 
 # .so.0 is for version 3.x, .so.1 is 4.x
-SONAME_CURRENT ?= 1
+SO_TARGET ?= libiniparser.so.1
 
 # Set RANLIB to ranlib on systems that require it (Sun OS < 4, Mac OSX)
 # RANLIB  = ranlib
@@ -56,22 +56,22 @@ SRCS = src/iniparser.c \
 OBJS = $(SRCS:.c=.o)
 
 
-default:	libiniparser.a libiniparser.so
+default:	libiniparser.a $(SO_TARGET)
 
 libiniparser.a:	$(OBJS)
 	$(QUIET_AR)$(AR) $(ARFLAGS) $@ $^
 	$(QUIET_RANLIB)$(RANLIB) $@
 
-libiniparser.so:	$(OBJS)
-	$(QUIET_LINK)$(SHLD) $(LDSHFLAGS) $(LDFLAGS) -o $@.$(SONAME_CURRENT) $(OBJS) \
-		-Wl,-soname=`basename $@`.$(SONAME_CURRENT)
+$(SO_TARGET):	$(OBJS)
+	$(QUIET_LINK)$(SHLD) $(LDSHFLAGS) $(LDFLAGS) -o $(SO_TARGET) $(OBJS) \
+		-Wl,-soname=`basename $(SO_TARGET)`
 
 clean:
 	$(RM) $(OBJS)
 	@(cd test ; $(MAKE) clean)
 
 veryclean:
-	$(RM) $(OBJS) libiniparser.a libiniparser.so*
+	$(RM) $(OBJS) libiniparser.a $(SO_TARGET)
 	rm -rf ./html ; mkdir html
 	cd example ; $(MAKE) veryclean
 	cd test ; $(MAKE) veryclean
@@ -79,7 +79,7 @@ veryclean:
 docs:
 	@(cd doc ; $(MAKE))
 	
-check: libiniparser.so
+check: $(SO_TARGET)
 	@(cd test ; $(MAKE))
 
 example: libiniparser.a
