Upstream: yes
From a915abf35a6e43d2a0cacf62f2b4a8432a2358d8 Mon Sep 17 00:00:00 2001
From: Steven Siloti <ssiloti@gmail.com>
Date: Tue, 23 Feb 2016 20:15:23 -0800
Subject: [PATCH] dnssec: only run tests that require gcrypt if we have it

---
 src/resolve/test-dnssec.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/resolve/test-dnssec.c b/src/resolve/test-dnssec.c
index a093d86..c9b5ffa 100644
--- a/src/resolve/test-dnssec.c
+++ b/src/resolve/test-dnssec.c
@@ -327,10 +327,12 @@ static void test_dnssec_nsec3_hash(void) {
 int main(int argc, char*argv[]) {
 
         test_dnssec_canonicalize();
+#ifdef HAVE_GCRYPT
         test_dnssec_verify_dns_key();
         test_dnssec_verify_rrset();
         test_dnssec_verify_rrset2();
         test_dnssec_nsec3_hash();
+#endif
 
         return 0;
 }
-- 
2.7.1

