From 92efa24947ccc88e4d54641eeedc32bd86434daa Mon Sep 17 00:00:00 2001
From: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
Date: Tue, 6 Sep 2016 14:51:57 +0200
Subject: [PATCH] random: don't block at startup because of lack of entropy

Signed-off-by: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
---
 random.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/random.c b/random.c
index 936121f..417efa4 100644
--- a/random.c
+++ b/random.c
@@ -567,7 +567,7 @@ fill_random_seed(uint32_t seed[DEFAULT_SEED_CNT])
 
     memset(seed, 0, DEFAULT_SEED_LEN);
 
-    fill_random_bytes(seed, DEFAULT_SEED_LEN, TRUE);
+    fill_random_bytes(seed, DEFAULT_SEED_LEN, FALSE);
 
     gettimeofday(&tv, 0);
     seed[0] ^= tv.tv_usec;
-- 
2.10.0

