Upstream: yes
From bdb29e1f5906dd8634340e54b605ede759e2e133 Mon Sep 17 00:00:00 2001
From: Wouter van Kesteren <woutershep@gmail.com>
Date: Mon, 23 Mar 2015 18:19:47 +0100
Subject: blacklist kernels instead of whitelist

Change-Id: I1249f3de77b79e8a1696cf844bb4a4f164691e8b

diff --git a/configure.ac b/configure.ac
index 650395f..66cb986 100644
--- a/configure.ac
+++ b/configure.ac
@@ -97,10 +97,11 @@ linux*)
 	AC_MSG_CHECKING([for supported kernel])
 	kernel_version=`uname -r | cut -d'-' -f1`
 	case "$kernel_version" in
-	3.*|2.6*)
-		AC_MSG_RESULT([$kernel_version])
+	0.*|1.*|2.0.*|2.1.*|2.2.*|2.3.*|2.4.*)
+		AC_MSG_RESULT([NO!])
+		AC_MSG_ERROR([Linux kernel version $kernel_version is not supported by sydbox])
 		;;
-	2.5*)
+	2.5.*)
 		kernel_minor_version=`echo $kernel_version | cut -d'.' -f3`
 		if test "$kernel_minor_version" -lt 46; then
 			AC_MSG_RESULT([NO!])
@@ -109,8 +110,7 @@ linux*)
 		AC_MSG_RESULT([$kernel_version])
 		;;
 	*)
-		AC_MSG_RESULT([NO!])
-		AC_MSG_ERROR([Linux kernel version $kernel_version is not supported by sydbox])
+		AC_MSG_RESULT([$kernel_version])
 		;;
 	esac
 	;;
-- 
cgit v0.10.2

