Index: configure
==================================================================
--- a/configure
+++ b/configure
@@ -234,11 +234,10 @@
 fi
 
 echo "Configuring dhcpcd for ... $OS"
 rm -f $CONFIG_H $CONFIG_MK
 echo "# $OS" >$CONFIG_MK
-echo "CPPFLAGS+=	-DHAVE_CONFIG_H" >>$CONFIG_MK
 echo "/* $OS */" >$CONFIG_H
 
 for x in SYSCONFDIR SBINDIR LIBDIR LIBEXECDIR DBDIR RUNDIR; do
 	eval v=\$$x
 	# Make files look nice for import
@@ -293,10 +292,12 @@
 fi
 if [ -n "$LDFLAGS" ]; then
 	echo "LDFLAGS=" >>$CONFIG_MK
 	echo "LDFLAGS+=	$LDFLAGS" >>$CONFIG_MK
 fi
+
+echo "CPPFLAGS+=	-DHAVE_CONFIG_H" >>$CONFIG_MK
 
 # NetBSD: Even if we build for $PREFIX, the clueless user might move us to /
 LDELF=/libexec/ld.elf_so
 if [ -e "$LDELF" ]; then
 	echo "Linking against $LDELF"

