Upstream: yes

From 204977fe3cd976d4fda5e6524d485a6279ab1b55 Mon Sep 17 00:00:00 2001
From: Alex Suykov <alex.suykov@gmail.com>
Date: Wed, 26 Aug 2015 13:30:49 +0300
Subject: [PATCH] threads should be built (as stubs) even without $usethreads

---
 cnf/configure_mods.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/cnf/configure_mods.sh b/cnf/configure_mods.sh
index 2c3950a..ffe0ebb 100644
--- a/cnf/configure_mods.sh
+++ b/cnf/configure_mods.sh
@@ -105,8 +105,6 @@ extonlyif VMS-Stdio "$osname" == "vms"
 extonlyif VMS-Filespec "$osname" == "vms"
 
 extonlyif Thread "$usethreads" == 'define'
-extonlyif threads "$usethreads" == 'define'
-extonlyif threads-shared "$usethreads" == 'define'
 
 for d in ext cpan dist; do
 	msg "Looking for extensions recursively under $d/"
