Author: Thomas Witt
Upstream: no
Reasen: last test needs network access

diff -ru a/perl-5.20.0/dist/IO/t/io_sel.t b/perl-5.20.0/dist/IO/t/io_sel.t
--- a/perl-5.20.0/dist/IO/t/io_sel.t	2014-05-26 15:34:19.000000000 +0200
+++ b/perl-5.20.0/dist/IO/t/io_sel.t	2014-06-02 23:30:38.615806854 +0200
@@ -3,7 +3,7 @@
 select(STDERR); $| = 1;
 select(STDOUT); $| = 1;
 
-print "1..27\n";
+print "1..23\n";
 
 use IO::Select 1.09;
 
@@ -129,23 +129,3 @@
 print "not " unless $w == 1 ;
 $w = 0 ;
 print "ok 23\n" ;
-
-{
-    # perl #75156 - test we can delete a closed handle
-    require IO::Socket::INET;
-    my $fh = IO::Socket::INET->new(
-      Listen => 5,
-    );
-    my $sel = IO::Select->new(\*STDIN);
-    $sel->add($fh);
-    my $oldbits = $sel->bits;
-    print "not " unless $sel->count == 2;
-    print "ok 24 - added socket\n";
-    close $fh;
-    print "not " unless $sel->remove($fh) == 1;
-    print "ok 25 - removed closed socket\n";
-    print "not " unless $sel->count == 1;
-    print "ok 26 - count() updated\n";
-    print "not " unless $sel->bits ne $oldbits;
-    print "ok 27 - bits() updated\n";
-}
